src/veins/modules/mobility/traci/TraCIScenarioManager.cc

```diff
@@ -480,8 +498,8 @@ void TraCIScenarioManager::finish() {
 	if (connection) {
 		TraCIBuffer buf = connection->query(CMD_CLOSE, TraCIBuffer());
 	}
-	while (hosts.begin() != hosts.end()) {
```

Comment:
Yes, I renamed `hosts` to `hostModules` to clarify what this member manages, i.e. the actual hosts' modules. The reason is that I also introduced another member holding the hosts' positions (for unequipped and equipped vehicles) and I thought naming it this way might help to prevent confusion (also for myself).